nix/docker.nix

```diff
@@ -105,31 +105,44 @@ let
     in ''
         elif [[ "$NETWORK" == "${env.name}" ]]; then
         ${if (env ? nodeConfig) then ''
+            echo "Connecting to network: ${env.name}"
             if [[ ! -z "''${EXTENDED}" ]] && [[ "''${EXTENDED}" == true ]]
             then
               exec ${dbSyncExtendedScript}
             else
               exec ${dbSyncScript}
             fi
+            echo "Cleaning up"
         ''
         else "echo db-sync not supported on ${env.name} ; exit 1"}
     '')));
     entry-point = writeScriptBin "entry-point" ''
       #!${runtimeShell}
+      mkdir -p /config
+      if [ ! -f /config/pgpass ]
+      then
+        echo "Generating PGPASS file"
```

Comment:
Logic looks good, but being a utility function for the service, I suggest hoisting this out into `/scripts/gen-pgpass-file.sh`, with the secrets directory passed in.

Reply:
Not required. Can always be refactored to later